bot/exts/moderation/voice_gate.py

```diff
@@ -28,10 +28,10 @@
 )
 
 MESSAGE_FIELD_MAP = {
-    "joined_at": f"been on the server for less than {GateConf.minimum_days_member} days",
+    "joined_at": f"been on the server for fewer than {GateConf.minimum_days_member} days",
     "voice_gate_blocked": "an active voice infraction",
-    "total_messages": f"sent less than {GateConf.minimum_messages} messages",
-    "activity_blocks": f"been active for fewer than {GateConf.minimum_activity_blocks} ten-minute blocks",
+    "total_messages": "have not been active enough on the server yet.",
+    "activity_blocks": "have not been active enough on the server yet.",
```

Comment:
I suggest the one I proposed for the activity blocks, and as mentioned we should get rid of the `total_messages` threshold (which will require other changes than just these messages).